packages/ag-grid-theme/package.json

```diff
@@ -14,8 +14,7 @@
     "/salt-ag-theme.css"
   ],
   "scripts": {
-    "build": "yarn node ./scripts/build.mjs",
-    "build-watch": "yarn node ./scripts/build.mjs --watch"
```

Comment:
Probably worth to have a watch for easy dev?

Reply:
This is not helpful as we're not using sass anymore